src/workflow/show_mnemonic.c

```diff
@@ -132,16 +132,12 @@ static bool _show_words(const char** words, uint8_t words_count)
 }
 
 typedef struct {
-    char* mnemonic;
-    // Keep len as mnemonic is tokenized using strtok, so util_cleanup_str() does not work anymore
-    // to clean the string.
-    size_t len;
+    char mnemonic[300];
```

Comment:
Why not 216?

> Should be at least 216 bytes (longest possible 24 word phrase plus null terminator).

Reply:
No harm in having a safety buffer, in case there is a miscalculation at 216 ;)